frontend_tests/node_tests/narrow.js

```diff
@@ -496,6 +523,32 @@ run_test("show_empty_narrow_message", ({mock_template}) => {
         ),
     );
 
+    people.add_active_user(ray);
+    set_filter([["group-pm-with", ray.email]]);
+    hide_all_empty_narrow_messages();
+    narrow_banner.show_empty_narrow_message();
+    assert.equal(people.verify_non_active_human_id(ray.user_id), false);
+    assert.equal(
+        $(".empty_feed_notice_main").html(),
+        empty_narrow_html(
+            "translated: You have no group private messages with Raymond yet.",
+            'translated HTML: Why not <a href="#" class="empty_feed_compose_private">start the conversation</a>?',
+        ),
+    );
```

Comment:
This block of code too seems extraneous for the same reason

Reply:
The same reason goes for this code snippet too.

Reply:
Here too, just remove the new code above this comment, not below

Reply:
Just remove the code block above, not below this comment